base/src/main/java/io/spine/logging/Logging.java

```diff
@@ -47,9 +47,11 @@
  *          distinguishable from the real code.
  */
 @SuppressWarnings({
-        "NewMethodNamingConvention", "PMD.MethodNamingConventions"
+        "NewMethodNamingConvention", "PMD.MethodNamingConventions",
         /* These methods are prefixed with underscore to highlight the fact that these methods
            are for logging, and to make them more visible in the real code. */
+        "FloggerSplitLogStatement" /* We use logging API directly for brevity.
```

Comment:
At this point, this error is suppressed in `javac-args.gradle`. Does the error pop up anyway?